third_party/d3js/README.md

```diff
@@ -0,0 +1,27 @@
+# d3js
+
+## URL
+
+https://github.com/d3/d3/releases
+
+## Version
+
+5.16.0
+
+## Commit
+
+[7244e45e68af39c519f76667a03039d5b24dd453](https://github.com/d3/d3/releases/tag/v5.16.0)
```

Comment:
If the commit points to the 5.16.0 tag, I think we can just remove the commit info, and have the version above being a link.

Reply:
👌